scripts/typecheck_tests.py

```diff
@@ -213,11 +213,13 @@
         'Unexpected keyword argument "headline__startswith" for "in_bulk" of "QuerySet"',
     ],
     'many_to_one': [
-        'Incompatible type for "parent" of "Child" (got "None", expected "Union[Parent, Combinable]")'
+        'Incompatible type for "parent" of "Child" (got "None", expected "Union[Parent, Combinable]")',
+        'Incompatible type for "parent" of "Child" (got "Child", expected "Union[Parent, Combinable]")'
     ],
     'model_meta': [
         '"object" has no attribute "items"',
-        '"Field" has no attribute "many_to_many"'
+        '"Field" has no attribute "many_to_many"',
+        re.compile(r'Definition of "(objects|_default_manager)" in base class "FirstParent" is incompatible with definition in base class "SecondParent"'),
```

Comment:
Could you move all those ignores to `__common__`, possibly with the comment with a reference to mypy issue/PR? 

Smth along the
```
re.compile(r'Definition of "(objects|_default_manager)" in base class "[A-Za-z0-9]+" is incompatible with definition in base class "[A-Za-z0-9]+"')
```

Reply:
Done 👍